test/format/util.c

```diff
@@ -165,15 +165,13 @@ fclose_and_clear(FILE **fpp)
 void
 timestamp_once(WT_SESSION *session)
 {
-    static const char *oldest_timestamp_str = "oldest_timestamp=";
     WT_CONNECTION *conn;
     WT_DECL_RET;
-    char buf[WT_TS_HEX_STRING_SIZE + 64];
+    char ts_hex_buf[WT_TS_HEX_STRING_SIZE];
+    char tscfg[64];
```

Comment:
Is this still enough? `WT_TS_HEX_STRING_SIZE` seems big.
I think if we're following the previous convention, it should be `64 + 2 * WT_TS_HEX_STRING_SIZE`.

Reply:
64 bytes should be enough, we use the same value in timestamp abort and other tests. In total I calculated that we need 55 bytes